tests/misc_env/install_prereq.py

```diff
@@ -130,3 +132,26 @@ def enable_rhel_rpms(ceph, distro_ver):
     for repo in repos:
         ceph.exec_command(
             sudo=True, cmd='subscription-manager repos --enable={r}'.format(r=repo), long_running=True)
+
+
+def registry_login(ceph, distro_ver):
+    ''' login to this registry 'registry.redhat.io' on all nodes
+        docker for RHEL 7.x and podman for RHEL 8.x'''
+    cdn_cred = get_cephci_config().get('cdn_credentials')
+    if cdn_cred:
+        user = cdn_cred.get('username')
+        pwd = cdn_cred.get('password')
+        if not (user and pwd):
+            return
```

Comment:
if you have credentials are missing and when they are needed for the scenario, how are you throwing error. you are just returning without any message.

Reply:
@rakeshgm sure I will handle that error